### ISSUE TITLE
Error on https://ruffle.rs/demo/

### ISSUE THREAD

**Issue by JetYuh** [ruffle-rs/ruffle#6575]
# Error Info
Error name: Error
Error message: panicked at 'already borrowed: BorrowMutError', core/src/avm2/object/script_object.rs:69:16
Error stack:
```
Error: panicked at 'already borrowed: BorrowMutError', core/src/avm2/object/script_object.rs:69:16
    at c.wbg.__wbg_new_df6e6ab7a65c4c4d (https://ruffle.rs/demo/966.index.js:1:7451)
    at std::sync::once::Once::call_once::{{closure}}::h3d23ff52b4e09547 (https://ruffle.rs/demo/f9a94783e5019f829e03.wasm:wasm-function[1071]:0x2bbf0d)
    at std::sync::once::Once::call_inner::h51a70f81ef530383 (https://ruffle.rs/demo/f9a94783e5019f829e03.wasm:wasm-function[1534]:0x33f6d4)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::ha9f849b489b96018 (https://ruffle.rs/demo/f9a94783e5019f829e03.wasm:wasm-function[5123]:0x4a08ac)
    at std::panicking::rust_panic_with_hook::h12b2f0179ff84c92 (https://ruffle.rs/demo/f9a94783e5019f829e03.wasm:wasm-function[3436]:0x4431dd)
    at core::panicking::panic_fmt::hf0d93ec1ac504cf9 (https://ruffle.rs/demo/f9a94783e5019f829e03.wasm:wasm-function[5430]:0x4b02f1)
    at core::result::unwrap_failed::hd5a60243cf946632 (https://ruffle.rs/demo/f9a94783e5019f829e03.wasm:wasm-function[4298]:0x486b4b)
    at <ruffle_core::avm2::object::Object as ruffle_core::avm2::object::TObject>::init_property::h7f708ba836c6fa56 (https://ruffle.rs/demo/f9a94783e5019f829e03.wasm:wasm-function[402]:0x10a513)
    at ruffle_core::avm2::activation::Activation::run_actions::h9dff52fe9060819b (https://ruffle.rs/demo/f9a94783e5019f829e03.wasm:wasm-function[350]:0x1f2d1)
    at ruffle_core::avm2::function::Executable::exec::h8464defc31ec7140 (https://ruffle.rs/demo/f9a94783e5019f829e03.wasm:wasm-function[844]:0x26226a)
```

# Player Info
Allows script access: false

# Page Info
Page URL: https://ruffle.rs/demo/

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.84 Safari/537.36
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2022-03-30
Channel: nightly
Built: 2022-03-30T00:15:16.448Z
Commit: a671d2977be582c1ed429eb5a75fa09eeec6a113
Is extension: false

**Comment by adrian17** [comment]
Can you please tell me what SWF you loaded that caused it?

**Comment by JetYuh** [comment]
https://drive.google.com/file/d/1CpGdrWKpQRSxK876zj7ImYb-SEgC2rUo/view

**Comment by Herschel** [comment]
The panic is fixed by #6579, although still needs better AS3 support for this to run.
